### PR TITLE
DashboardScene: Add rows keyboard shortcuts

### PR DESCRIPTION
**Pull request by dprokop**
Closes https://github.com/grafana/hyperion-planning/issues/102

### CODE REVIEW

File: public/app/features/dashboard-scene/scene/DashboardScene.tsx [hunk 1]
```diff
@@ -877,6 +877,40 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
...
+
+    sceneGridLayout.state.children.forEach((child) => {
+      if (!(child instanceof SceneGridRow)) {
+        return;
+      }
+      if (child.state.isCollapsed) {
+        sceneGridLayout.toggleRow(child);
+      }
+    });
+  }
```
Comment (ivanortegaalba) [inline public/app/features/dashboard-scene/scene/DashboardScene.tsx:913]: I'd add some tests to `DashboardScene.test.tsx`. Apart from that, looks good to me!

File: public/app/features/dashboard-scene/scene/keyboardShortcuts.ts [hunk 2]
```diff
@@ -167,12 +167,26 @@ export function setupKeyboardShortcuts(scene: DashboardScene) {
         }
       }),
     });
+
+    // collapse all rows
+    keybindings.addBinding({
```
[the remark this answers is not in the document]
Comment (dprokop) [reply public/app/features/dashboard-scene/scene/keyboardShortcuts.ts:172]: 👍

[the remark this answers is not in the document]
Comment (dprokop) [reply public/app/features/dashboard-scene/scene/keyboardShortcuts.ts:172]: @ivanortegaalba  - i've added a new suite for dashboard keybindings

**Review by ivanortegaalba — APPROVED** [review]
Thanks for considering the feedback Dominik! LGTM!

**Pull request merged**